taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=240, unique=True)
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=240, unique=True)
+    country = models.CharField(max_length=240)
+
+    def __str__(self):
+        return self.name
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=240)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="Manufacturer"
+    )
+    drivers = models.ForeignKey(
+        Driver,
+        on_delete=models.SET_NULL,
+        null=True,
+        related_name="Drivers"
```

Comment:
The `related_name` is picked up incorrectly for both fields, the related name should represent the way how model objects would be retrieved from the side of the related model, e.g. you would get all cars that relate to certain driver by referring to `cars` from the driver, not `Drivers`. Related names should be lowercase and preferably in a plural form. [Here](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.ForeignKey.related_name) you can read more extensive documentation on this topic. Use `"cars"` as a `related_name` for both `drivers` and `manufacturer` fields.